src/psbt.h

```diff
@@ -307,7 +307,7 @@ struct PSBTOutput
 /** A version of CTransaction with the PSBT format*/
 struct PartiallySignedTransaction
 {
-    boost::optional<CMutableTransaction> tx;
+    Optional<CMutableTransaction> tx;
```

Comment:
imo, 17316 shouldn't be marked partial (even though I see there was once section where we excluded a change)

Reply:
Hello @PastaPastaPasta thanks for the comments, I have removed the Partial tag from those two commits,